agent-control/src/agent_control/defaults.rs

```diff
@@ -32,6 +32,7 @@ pub const HOST_NAME_ATTRIBUTE_KEY: &str = opentelemetry_semantic_conventions::at
 pub const CLUSTER_NAME_ATTRIBUTE_KEY: &str = "cluster.name";
 pub const HOST_ID_ATTRIBUTE_KEY: &str = opentelemetry_semantic_conventions::attribute::HOST_ID;
 pub const FLEET_ID_ATTRIBUTE_KEY: &str = "fleet.guid";
+pub const OS_ATTRIBUTE_KEY: &str = "os.type";
```

Comment:
would make sense to link to here or specify the variants to avoid miss used in the future ? https://opentelemetry.io/docs/specs/semconv/registry/attributes/os/#os-type